subprojects/plugins/src/main/java/org/gradle/jvm/application/tasks/CreateStartScripts.java

```diff
@@ -225,6 +225,7 @@ public void setDefaultJvmOpts(@Nullable Iterable<String> defaultJvmOpts) {
     /**
      * The application's name.
      */
+    @Nullable
```

Comment:
:x: the corresponding setter should have the same nullability as the getter otherwise setting a value in Kotlin will require using `setApplicationName(some)`

Reply:
Good to know. Thanks.

Reply:
Fixed.